.chloggen/codeboten_obsreport-dep.yaml

```diff
@@ -0,0 +1,29 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: deprecation
+
+# The name of the component, or a single word describing the area of concern, (e.g. otlpreceiver)
+component: obsreport
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: "Deprecating `obsreport.Exporter`, `obsreport.ExporterSettings`, `obsreport.NewExporter`" 
+
+# One or more tracking issues or pull requests related to the change
+issues: [8492]
+
+# (Optional) One or more lines of additional information to render under the primary note.
+# These lines will be padded with 2 spaces and then inserted directly into the document.
+# Use pipe (|) for multiline entries.
+subtext: |
+  These deprecated methods/structs have been moved to exporterhelper:
+  - `obsreport.Exporter` -> `exporterhelper.Exporter`
+  - `obsreport.ExporterSettings` -> `exporterhelper.Settings`
+  - `obsreport.NewExporter` -> `exporterhelper.New`
```

Comment:
I think the new API naming is confusing. It still provides the functionality for telemetry reporting but the names don't reflect it anymore. Should we move it exporterhelper/obsreport instead?

Reply:
See https://github.com/open-telemetry/opentelemetry-collector/pull/8499